README.md

```diff
@@ -6,8 +6,32 @@ The main repo for Stage Whisper, our easy to use AI transcriber, powered by Open
 
 Stage Whisper consists of two connected components:
 
-* A Node/Electron-powered interface
 * A Python backend that interfaces with OpenAI's Whisper library
+* A Node/Electron-powered interface
+
+### Prerequisites
+
+You will need the following installed on your machine to develop Stage Whisper:
+
+* [Python 3.x](https://www.python.org/)
+* [Node](https://nodejs.org/en/)
+* [Rust](https://www.rust-lang.org/)
+* [ffmpeg](https://ffmpeg.org/)
+* [Poetry](https://python-poetry.org/)
```

Comment:
Don't think you need poetry anymore. Or at the very least, not for the electron frontend

Reply:
Perhaps we can mention that this step is only required for the backend you can install Whisper separately

Reply:
Eventually the ability to install whisper separately will go away when we successfully link the backend and electron app. (The backend is going to do more than whisper cli ultimately.)

It’s true this isn’t necessary now, but it will be soon. I could separate out which reqs are backend only and which reqs are electron only but I worry a bit about adding flowchart like logic to the readme 😅

Reply:
I added some clarification to which components are required for the backend and which for electron, which should help.